# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/debug/breakpoint-11.js
# Commit: 73c33bf110d6
# Full Hash: 73c33bf110d6034266cbe615b9e018476a128d63
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 2
# Description:
#   Bug 1600204 - Part 2: Remove Debugger.Frame.prototype.generator since it is already on script. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55232
# ==============================================================================

diff -r d70ec1e79887 -r 73c33bf110d6 js/src/jit-test/tests/debug/breakpoint-11.js
--- a/js/src/jit-test/tests/debug/breakpoint-11.js	Fri Dec 06 02:29:18 2019 +0000
+++ b/js/src/jit-test/tests/debug/breakpoint-11.js	Fri Dec 06 02:57:20 2019 +0000
@@ -6,8 +6,6 @@
 var dbg = Debugger(g);
 dbg.onDebuggerStatement = function (frame) {
     function hit(frame) {
-        assertEq(frame.generator, true);
-        assertEq(frame.older.generator, true);
         frame.older.eval("q += 16");
     }
 